_assets/scripts/install_deps.sh

```diff
@@ -7,7 +7,7 @@ elif [[ -x $(command -v pacman) ]]; then
 elif [[ -x $(command -v brew) ]]; then
   brew install protobuf jq
 elif [[ -x $(command -v nix-env) ]]; then
-  nix-env -iA nixos.protobuf3_17
+  nix-env -iA nixos.protobuf3_20
```

Comment:
Completely unnecessary, this script and its references can be removed. Everything is provided by Nix.